wiki/Music_theory/Downbeat/de.md

```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+tags:
+  - downbeats
+  - first beat
+  - erste Note
+---
+
+# Downbeat
+
+Der **Downbeat** ist die erste Note in einem Takt. Er ist auf der Zeitleiste des [Beatmap-Editors](/wiki/Client/Beatmap_editor) als große weiße Striche erkennbar. Sein Name ist von der Abwärtsbewegung des Dirigentenstabs bei der ersten Note jedes Takts abgeleitet. In der Regel ist er im rhythmischen Zyklus am meisten betont.
```

Comment:
The closest I can think of for measured music is `Musiktheorie` or `Notenschrift`. Since these terms are redundant in the article, I left it out.

Reply:
I think what is meant by `measured music` is `music that bases its rhythm on measures`, or something similar to it. So maybe `taktbasierte Musik` would be a decent translation.
But honestly, that might be more confusing than omitting it. So I'm fine with this too